modules/control/control_component.cc

```diff
@@ -277,6 +277,10 @@ Status ControlComponent::ProduceControlCommand(
 
 bool ControlComponent::Proc() {
   const auto start_time = Clock::Now();
+  const double start_system_timestamp =
```

Comment:
suggest "const auto" to keep consistent with the others

Reply:
Fixed. Thank you.